### PR TITLE
Updates to Chef server 12

### PR DESCRIPTION
**Pull request by jjasghar**
- Updated from Opscode to Chef
- Added `_status` method
- Added `containers` method

### CODE REVIEW

File: methods/_status.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+exports.methods = function(config){
+
+    var http_methods = require([__dirname, "..", "http_methods"].join("/")).http_methods(config);
+
+    return {
+        // https://docs.chef.io/api_chef_server.html#status
+        getStatus: function(fn){
+            http_methods.get([config.host_url, "/_status"].join("/"), null, function(err, response){
```
Comment (normanjoyner) [inline methods/_status.js]: extra slash here as array is being joined with a slash already

Comment (jjasghar) [inline methods/_status.js]: Awesome, update here soon.

**Comment by jjasghar** [comment]
Any chance i can get this merged? I'd love to use these changes for a plugin i'm trying to get out. :metal:

**Comment by normanjoyner** [comment]
ahh sorry about that, I somehow lost track of this PR. I will review and get it merged ASAP!

**Comment by jjasghar** [comment]
Awesome thanks!

**Comment by normanjoyner** [comment]
@jjasghar landed in [0.6.0](https://github.com/normanjoyner/chef-api/releases/tag/0.6.0)

**Comment by jjasghar** [comment]
Rock on thanks!

**Pull request merged**